binding/nodejs/index.test.js

```diff
@@ -26,9 +26,6 @@ const MODEL_PATH_ES = "../../lib/common/porcupine_params_es.pv";
 const MODEL_PATH_FR = "../../lib/common/porcupine_params_fr.pv";
 
 const SENSITIVITIES_1 = [0.5];
```

Comment:
can you just rename this like `DEFAULT_SENSITIVITY_ARRAY` or something? since we're not going to create anymore.